### PR TITLE
Changes to automate integration tests

### PR DESCRIPTION
**Pull request by kbalk**
Use tardigrade-ci's ability to run automated integration tests against a mock AWS stack.

### CODE REVIEW

**Comment by kbalk** [comment]
@lorengordon I'm wondering if it's one of my changes that's causing it to hang up.  I can recreate this branch from scratch and see what changes cause the problem.

File: .travis.yml [hunk 1]
```diff
@@ -1,20 +1,36 @@
...
+      script: make docker/run target=lint
+    - stage: test
+      name: Apply Terraform test configs in mockstack
+      install:
+        - make docker-compose/install
+        - make mockstack/up
+      script: make mockstack/pytest
+      after_script:
+        - make mockstack/clean
+   - stage: deploy
```
Comment (lorengordon) [inline .travis.yml:33]: Oh geez, here it is. Bad yaml...
```suggestion
    - stage: deploy
```

Comment (lorengordon) [reply .travis.yml:33]: I found it by going to the "requests" page in travis-ci, and seeing the "Could not parse" message... https://travis-ci.com/github/plus3it/terraform-aws-tardigrade-config/requests

Comment (kbalk) [reply .travis.yml:33]: Good eye!  Maybe we should add that to the lint steps at some future date.

Comment (lorengordon) [reply .travis.yml]: I was thinking something similar, but I'm not sure how to have the CI validate a parsing error in the CI's own config... It has to parse the config to run the test in the first place... But we could at least provide a target that we could run _locally_ to validate yaml though. Or even use the [`travis` cli to validate its config](https://support.travis-ci.com/hc/en-us/articles/115002904174-Validating-travis-yml-files).

Comment (kbalk) [reply .travis.yml]: Good to know.  I was worried how to debug the issue.  I had reviewed my additions to .travis.yaml, but my eye stopped at the end of the 'test' stage additions, not the line just beyond.

**Review by lorengordon — APPROVED** [review]
This one validates that terraform is properly configured and able to hit the moto server port also. 🚀

**Pull request merged**